mlir/include/mlir/Dialect/SPIRV/IR/TargetAndABI.h

```diff
@@ -89,7 +89,8 @@ StringRef getEntryPointABIAttrName();
 /// Gets the EntryPointABIAttr given its fields.
 EntryPointABIAttr getEntryPointABIAttr(MLIRContext *context,
```

Comment:
Could you add a line of explanation for target width and link to the relevant section of the spirv spec? 'Target width' is a bit of an unfortunate name because it sounds very generic.

Reply:
There is no specific section to describe `target width`. It appears in the `Execution Mode`. I have added the link to execution mode and added some description about `target width` in the execution mode.